electron-test/tsconfig.json

```diff
@@ -19,7 +19,6 @@
         "allowImportingTsExtensions": true,
         "resolveJsonModule": true,
         "isolatedModules": true,
-        "strict": true,
```

Comment:
@axi92 This affects only the electron test project not the MQTTjs project so not a big deal IMO

Reply:
Done. Please check it.